src/main/java/org/fundacionjala/coding/abel/movies/Customer.java

```diff
@@ -0,0 +1,102 @@
+package org.fundacionjala.coding.abel.movies;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Unknown on 5/26/2017.
+ */
+public class Customer {
+
+    private String name;
+    private List<Rental> rentals;
+    private double totalAmount;
+    private int frequentRenterPoints;
+    private static final List<MovieType> MOVIE_TYPES = Arrays.asList(
+            new MovieType(Movie.REGULAR, new Regular()),
+            new MovieType(Movie.NEW_RELEASE, new NewRelease()),
+            new MovieType(Movie.CHILDREN, new Children())
+    );
+
+    /**
+     * This is the default constructor.
+     * @param name String for customers name.
+     */
+    public Customer(String name) {
+        this.name = name;
+        rentals = new ArrayList<>();
+    }
+
+    /**
+     * Method for add Rentals.
+     * @param rental Instance of Rental.
+     */
+    public void addRental(Rental rental) {
+        rentals.add(rental);
+    }
+
+    /**
+     * Getter for Customer Name.
+     * @return Name of Customer.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * .
+     * @return Statement.
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things, it is calculating the totalAmount, totalFrequentRenterPoints and it is printing.
Please split it.